standard/interfaces.md

```diff
@@ -718,6 +718,10 @@ When an interface is part of a namespace, a qualified interface member name can
 
 Interfaces may be implemented by classes and structs. To indicate that a class or struct directly implements an interface, the interface is included in the base class list of the class or struct.
 
+A class or struct `C` that implements an interface `I` must provide or inherit an implementation for every member declared in `I`. Public members of `I` may be defined in public members of `C`. Non-public members declared in `I` that are accessible in `C` may be defined in `C` using explicit interface implmentation (§19.6.2). Non-accessible members, such as `private` members of `I` can't be defined in `C`.
```

Comment:
This starts off with “must provide or inherit an implementation for *every* member declared in `I`” (emphasis added), then says how some of these members may be given implementations, and ends with “Non-accessible members, such as `private` members of `I` can't be defined in `C`” – should this last statement say something along the lines that in this case an implementation must be inherited (or something else)?

Reply:
This might be better handled in another PR.  The rules in this PR state that an interface that declares a private member must provide a definition for that member. It's inaccessible to any derived interface or class, or struct that implements the interface.

It might be that this can just remove removed. Otherwise, it should relate to those rules as well.